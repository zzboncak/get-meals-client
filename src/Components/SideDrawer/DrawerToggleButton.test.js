import React from 'react';
import ReactDOM from 'react-dom';
import DrawerToggleButton from './DrawerToggleButton';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

describe(`DrawerToggleButton component`, () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');
      ReactDOM.render(<DrawerToggleButton />, div);
      ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the component as expected', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
            <DrawerToggleButton />
        </BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})
