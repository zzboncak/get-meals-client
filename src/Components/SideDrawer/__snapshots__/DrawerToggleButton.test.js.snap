// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerToggleButton component renders the component as expected 1`] = `
<button
  className="toggle-button"
>
  <div
    className="toggle-button__line"
  />
  <div
    className="toggle-button__line"
  />
  <div
    className="toggle-button__line"
  />
</button>
`;
